dmscripts/generate_database_data.py

```diff
@@ -73,7 +73,7 @@
             'unitSingular': 'service'
         }
     ],
-    'status': 'live',
```

Comment:
If we're not using this, does it still need to be here?

Reply:
No, it does not need to be there. However, `G_CLOUD_FRAMEWORK` represents a complete json representation of the object as the API returns it. If we remove that one key, that won't be true anymore, so it is not clear what `G_CLOUD_FRAMEWORK` anymore. That's the only reason why I kept it.
Do you think it is a strong enough reason to keep that @bjgill?

Reply:
That's a good reason to keep it, but should we keep it as a valid value in that case?

Reply:
That's reasonable. Fixed in https://github.com/alphagov/digitalmarketplace-scripts/pull/707/commits/a089913c04dc2f4753e4fbb6050cdbe875258759